backend/src/personnel/dto/personnel-experiences.dto.ts

```diff
@@ -7,7 +7,14 @@ export class PersonnelExperienceDTO {
     required: true,
     example: 1,
   })
-  id: number;
+  functionId?: number;
+
+  @ApiProperty({
+    description: 'Id of Function',
+    required: true,
+    example: 1,
+  })
+  id?: number;
```

Comment:
This is kinda strange, but if its the only way to make it work, can you comment that further?

Reply:
Not the only way, just the only way without making changes elsewhere. I can put a TODO to refactor post MVP?